modules/1_prepare/versions.tf

```diff
@@ -29,5 +29,5 @@ terraform {
       version = "~> 2.1"
     }
   }
-  required_version = "~> 0.13.0"
+  required_version = "~> 1.0.8"
```

Comment:
Why 1.0.8? If I am not wrong there is no major changes and it should work with any version greater than 1.0.0.
Also, we should use `>=` to auto support `1.1.0` which is under beta currently.

Reply:
Thanks. Updated the change.